src/specifications/profile-resolution/profile-resolution-specml.xml

```diff
@@ -3,1772 +3,1370 @@
-              <tgt>responsible-party</tgt> elements. The target is schema valid, and the new
-              <tgt>link</tgt> is given before copies of <src>link</src> elements already present</p>
-          <p>The new link should have two flags (attributes):</p>
-          <ul>
-            <li>
-              <p><tgt>@rel</tgt> has the value <q>resolution-source</q></p>
-            </li>
-            <li>
-              <p><tgt>@href</tgt> provides a URL or URI address for the source profile</p>
-            </li>
-          </ul>
-          <tagging whose="source_profile">&lt;metadata>
-  &lt;title>Test profile&lt;/title>
-  &lt;last-modified>2019-12-03T11:36:32.284-05:00&lt;/last-modified>
-  &lt;version>1.0&lt;/version>
-  &lt;oscal-version>1.0-MR2&lt;/oscal-version>
-  &lt;party>
-    &lt;org>
-      &lt;org-name>Standing Committee&lt;/org-name>
-    &lt;/org>
-  &lt;/party>
-&lt;/metadata></tagging>
-          <tagging whose="target_catalog">&lt;metadata>
-  &lt;title>Test profile - RESOLUTION RESULT&lt;/title>
-  &lt;last-modified><hi>{ timestamp of profile resolution runtime }</hi>&lt;/last-modified>
-  &lt;version>1.0&lt;/version>
-  &lt;oscal-version>1.0-MR2&lt;/oscal-version>
-  &lt;link rel="resolution-source" href="<hi>{ URI to resolution source data instance }</hi>">Test profile&lt;/link>
-  &lt;party>
-    &lt;org>
-      &lt;org-name>Standing Committee&lt;/org-name>
-    &lt;/org>
-  &lt;/party>
-&lt;/metadata></tagging>
-        </mapping>
+      <section id="why-a-catalog">
+        <!-- TODO: Needs more in-depth editing pass -->
+        <head>Why resolve a profile as a catalog</head>
+        <p>A profile is a representation of a delta, a <q>here to there</q>. As such, a profile
+          might be conceived of as a recipe or instruction set – a sequence of steps or procedures –
+          whereby to create an OSCAL catalog (set of controls describing security requirements) from
+          an OSCAL catalog (set of controls describing security requirements). The primary use case
+          for this is when we need to make a new catalog to apply to a system or set of systems,
+          starting from a canonical, received, mandated or standard catalog with a more general
+          scope.</p>
+        <p>This is because although catalogs - for example, the controls described in NIST SP 800-53
+          - are published, available for use, and even standardized, they can never be used exactly
+          in the form in which they are acquired. They must be taken from the packaging and placed
+          into operational context before they mean anything. This typically requires that a few
+          operations over <q>raw catalogs</q> be supported by a profiling mechanism. We must be able
+          to select controls; we must be able to arrange them in order (or not); and we must be able
+          to qualify or condition them by setting parameters defined for them and making adjustments
+          to them. </p>
```

Comment:
It may be also necessary to tailor controls by adding or removing information to provide more context for a given use of the controls.

Reply:
You talk about making adjustments. You should call this tailoring.